Fixes for the Redis Cloud quickstart guide (#2911)
RedisLabs/redislabs-docs: content/rc/rc-quickstart.md

To create a new account with a free subscription and database:

1. Navigate to the [Sign up](https://redis.com/try-free/) page.

1. There are two options available to get started with Redis Cloud:

>    * Enter your information in the form and select **Get Started**

Reviewer: ```suggestion
   * Enter your information in the form and select **Get Started**.
```

>    * Sign up with **Google** or **Github**.

Reviewer 2: ```suggestion
   * Sign up with **Google** or **Github**.
```
If at all possible, we should make these links to the relevant google and github pages.

Reviewer: I'm not sure this is possible. If I click the social login buttons, it just brings me to a social login screen that's specific to my accounts.

>    * Sign up with **Google** or **Github**

Reviewer: ```suggestion
   * Sign up with **Google** or **Github**.
```

1. Once you have completed step 2, you should receive an activation email from Redis. In the activation email, select **Activate account**. You will be forwarded to the **Get Started** page.

    {{<image filename="images/rc/quickstart-get-started.png" width="75%" alt="Dialog to create your free subscription." >}}{{< /image >}}

>    * Choose your role

Reviewer: ```suggestion
    1. Choose your role from the **Role** drop-down.
```

>    * Choose who you work for

Reviewer: ```suggestion
    1. Choose your company size from the **myself/company** drop-down. If you are working on a personal project, select **myself**.
```

>    * Choose your use case

Reviewer: ```suggestion
    1. Select your Redis use case from the **use case** drop-down.
```

>    * Choose your client

Reviewer: ```suggestion
     1. Choose your client language from the **client** drop-down.
```

>     Click **Create Free Database**. You also have the option to **Create a custom database** if necessary.

Reviewer: ```suggestion
    Click **Create Free Database** to move on.
```
We talk about choosing **create a custom database** in the note two steps down. I don't think we need to talk about creating a custom database here.

    {{<image filename="images/rc/quickstart-database-overview.png" width="75%" alt="Configuration tab showing details of your new database." >}}{{< /image >}}

> 1.  Confirm your database is active  

Reviewer: ```suggestion
1.  
```
I agree that there needs to be a number here, but this step doesn't need a heading and could just start with the next line.

    
    In the upper corner, an icon shows the current status of the database.  If the icon shows an orange clock, this means your database is still being created and its status is _pending_.

       ![Pending status icon](/images/rc/icon-database-update-status-pending.png#no-click "Pending database status") &nbsp; ![Active status icon](/images/rc/icon-database-update-status-active.png#no-click "Active database status")

       Once the database has been created, it becomes _active_ and the status indicator switches to a green circle containing a checkmark.  

    Admin console operations are asynchronous; they operate [in the background]({{< relref "/rc/api/get-started/process-lifecycle.md" >}}).  You can continue to use the admin console for other tasks, but pending resources aren't available until they're active.

    When your new database becomes active, you're ready to connect to it.

## Connect to a database

> At this point, you're viewing the **Configuration** details for your new database. Scroll to the **Security** section of the page.

Reviewer: ```suggestion
At this point, you're viewing the **Configuration** details for your new database. Go to the **Security** section of the page.
```

The connection wizard provides the following database connection methods:

- [RedisInsight](https://redis.com/redis-enterprise/redis-insight/)

- [`redis-cli`]({{<relref "/rs/references/cli-utilities/redis-cli/">}}) utility

- [Redis client](https://redis.io/clients) for your preferred programming language

{{<image filename="images/rc/connection-wizard.png" alt="The connection wizard." >}}{{< /image >}}

### RedisInsight{#using-redisinsight}

RedisInsight is a free Redis GUI that is available for MacOS, Windows, and Linux.

1. In the connection wizard, under **RedisInsight**, select your operating system from the **Download RedisInsight** menu.

1. Select **Download** to download RedisInsight.

> 1. [Install RedisInsight]({{< relref "/ri/installing/install-redis-desktop" >}}).

Reviewer: ```suggestion
1. [Install RedisInsight](https://redis.io/docs/connect/insight/).
```

> 1. Once installed, select **Open with RedisInsight**

Reviewer: ```suggestion
1. Once installed, select **Open with RedisInsight**.
```

> 1. A popuup appears asking you if you wish to open the link with the RedisInsight application. Select **Open RedisInsight**

Reviewer: ```suggestion
1. A pop-up asks if you wish to open the link with RedisInsight. Select **Open RedisInsight** to connect to your database with RedisInsight.
```

> 1. You are automatically connected to your database

Reviewer: ```suggestion

```

> See the [RedisInsight documentation]({{< relref "/ri/_index.md" >}}) for more information.

Reviewer: ```suggestion
See the [RedisInsight documentation](https://redis.io/docs/connect/insight/) for more information.
```

### Redis client{#using-redis-client}

A Redis client is a software library or tool that enables applications to interact with a Redis server. Each client has its own syntax and installation process. For help with a specific client, see the client's documentation.

The connection wizard provides code snippets to connect to your database with the following programming languages:

- node.js using [node-redis](https://github.com/redis/node-redis/blob/master/README.md)
- .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
- Python using [redis-py](https://github.com/redis/redis-py#redis-py)
- Java using [Jedis](https://github.com/redis/jedis#jedis)

{{<image filename="images/rc/connection-wizard-clients.png" alt="The connection wizard clients." >}}{{< /image >}}

See the [client list](https://redis.io/docs/clients/) to view all Redis clients by language.

> #### Code example (Python)

Reviewer: @dmaier-redislabs - Should we get rid of this section entirely? See DOC-2510

1. Install the Redis client if it is not already installed.

    ```sh
    $ sudo pip install redis
    ```

2. In the connection wizard, under **Redis Client**, select Python from the **Select your client** menu.

3. Select **Copy** to copy the connection code for your database.

4. Create a python file ```vi example_redis.py```

5. Add the copied code to your Python program.

    ```
    import redis

    r = redis.Redis(
    host='<HOST>',
    port=<PORT>,
    password='<PASSWORD>')
    ```
   
6. Add the following code to your Python program.

    ```
    r.set('hello', 'world')
    print(r.get('hello'))
    ```

7. Run the program.

    ```sh
    $ python example_redis.py
    b'world'
    ```

### redis-cli (via Docker){#using-rediscli}

The [`redis-cli`]({{<relref "/rs/references/cli-utilities/redis-cli/">}}) utility is installed when you install Redis.  It provides a command-line interface that lets you work with your database using core [Redis commands](https://redis.io/commands/).

[Docker](https://www.docker.com/) provides a convenient way to run `redis-cli` without the full installation experience.

> Run the following commands to create a Redis Docker container and connect to your database with `redis-cli`:

Reviewer: @dmaier-redislabs Same with this section (and same ticket - DOC-2510) - do we have somewhere else we talk about installing Redis with docker? redis.io doesn't have anything about docker in the [Install Redis](https://redis.io/docs/install/install-redis/) section.

1.  Download the `redis` Docker image:

    ``` sh
    $ docker pull redis
    ```
1.  Start a container created from the image:

    ``` sh
    $ docker run -d --name redis1 redis
    ```

> 1.  Connect to a bash prompt running in the container:

Reviewer: ```suggestion
1.  Connect to a shell prompt running in the container:
```
